src/Std/Time/DateTime/PlainDateTime.lean

```diff
@@ -70,11 +70,11 @@ def ofTimestampAssumingUTC (stamp : Timestamp) : PlainDateTime := Id.run do
       let h := rem.truncateBottom (Int.not_le.mp h)
       (h, rawDays)
 
-  let mut quadracentennialCycles := days / daysPer400Y;
-  let mut remDays := days.val % daysPer400Y.val;
+  let mut quadracentennialCycles := days.val / daysPer400Y;
+  let mut remDays := days.val % daysPer400Y;
```

Comment:
nit: Adding a `Mod` operation for `UnitVal` would be well-typed, and would clean this up a little.